client/platform/web-girder/views/AnnotationTags.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import {
+  defineComponent, ref,
+} from '@vue/composition-api';
+import {
+  useAnnotationTag,
+  useAnnotationTags,
+  useHandler,
+} from 'vue-media-annotator/provides';
+
+export default defineComponent({
+  name: 'AnnotationTags',
+  description: 'Annotation Tags',
+
+  setup() {
+    const currentTag = useAnnotationTag();
+    const tags = useAnnotationTags();
+    const newTag = ref('');
+    const validForm = ref(false);
+    const { save, tagChange } = useHandler();
+    const selectedTag = (tag: string) => {
+      tagChange(tag);
+    };
+
+    const addTag = async () => {
+      await save(newTag.value);
+      tagChange(newTag.value);
+    };
+
+    return {
+      currentTag,
+      selectedTag,
+      tags,
+      newTag,
+      addTag,
+      validForm,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <p> Available Tags</p>
+    <v-list>
+      <v-list-item
+        v-for="tag in tags"
+        :key="tag"
+      >
+        <v-chip
+          outlined
+          :color="tag === currentTag || (!currentTag && tag === 'default') ? 'cyan' : 'white'"
+          @click="selectedTag(tag)"
+        >
+          {{ tag }}
+        </v-chip>
+      </v-list-item>
+    </v-list>
+    <v-divider />
+    <v-expansion-panels>
+      <v-expansion-panel>
+        <v-expansion-panel-header> Add New Tag</v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <p>
+            Add a new Tag with a custom name that can be used to reference it in the future.
+            'default' is a reserved tag which can't be used.
+            Adding a new tag will tag the current annotations and copy them to the new tag.
```

Comment:
I wonder if it should be an option to add current annotations to a tag. I think I would expect the initial annotations to belong to `default`. I am thinking of `default` as ground truth, is that an incorrect thought?

I would expect everything after to be part of the new tag but not _necessarily_ everything before.

Reply:
So if you have no annotations at all there is no tag.  Once you start adding tags the previous annotations had an empty tag I named 'default'.  'default' can be whatever you want annotation for you annotations but it's the tag that pipelines/training will run on.  

If you update the annotations in tag and click save it will save to the currently selected tag.  I didn't implement anything for transferring annotations between tags except in the case of generating a brand new tag.